src/neuroconv/datainterfaces/ecephys/edf/edfdatainterface.py

```diff
@@ -36,15 +53,24 @@ def __init__(self, file_path: FilePath, verbose: bool = True, es_key: str = "Ele
         verbose : bool, default: True
             Allows verbose.
         es_key : str, default: "ElectricalSeries"
+            Key for the ElectricalSeries metadata
+        channels_to_skip : list, default: None
+            Channels to skip when adding the data to the nwbfile. These parameter can be used to skip non-neural
+            channels that are present in the EDF file.
+
         """
         get_package(
             package_name="pyedflib",
-            excluded_platforms_and_python_versions=dict(darwin=dict(arm=["3.8", "3.9"])),
+            excluded_platforms_and_python_versions=dict(darwin=dict(arm=["3.9"])),
```

Comment:
why are you changing this?

Reply:
We no longer support python 3.8, unecessary change though, reverting.

Reply:
oh, right, I forgot about that. It's fine to keep it, I was just wondering.

Reply:
Ok.